root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Aspx/Framework/myOKMessageDialog.aspx

```diff
@@ -5,28 +5,51 @@
 <!DOCTYPE html>
 <html>
     <head id="Head1" runat="server">
-        <title>myOKMessageBox</title>
-    </head>
-    <script type="text/javascript">
-        <!--
-        
-        // 戻り値
-        // 初期戻り値(-1)は、Dialog右上の[×]で閉じた場合に戻る値
-        // [OK]Button押下時点で、この変数値を書き換える
-        var dialogreturn = -1;
+        <title>OK Message Dialog</title>
+        <asp:PlaceHolder runat="server">
+            <%: Scripts.Render("~/bundles/modernizr") %>
+        </asp:PlaceHolder>
+        <webopt:bundlereference runat="server" path="~/bundles/css" />
+
+        <script type="text/javascript">
+            // Resolve URL in the javascript
+            var baseUrl = "<%= this.ResolveUrl("~/") %>";
+
+            // 初期戻り値(-1)は、Dialog右上の[×]で閉じた場合に戻る値
+            var dialogreturn = -1;
         
-        // [OK]Button押下
-        // 戻り値を 1 に書き換えた上で、このDialogを閉じる
-        function onOK(){
-            window.returnValue = 1;
-            window.close();
-        }
+            // 開始処理
+            function Document_OnLoad() {
+                Fx_WhichBrowser();
+
+                // 擬似ダイアログの場合のtitle設定
+                if (window.dialogArguments === null || window.dialogArguments === undefined) {
+                    document.getElementById("PseudoDialogTitle").innerText = document.title;
+                }
+                else {
+                    document.getElementById("PseudoDialogTitle").style.display = "none";
+                }
+            }
+
+            // [OK]Button押下時点で、戻り値を 1 に書き換えた上で、このDialogを閉じる
+            function onOK() {
+                if (Browser_IsIE) {
+                    window.returnValue = 1;
+                    window.close();
+                }
+                else {
+                    window.parent.Fx_CallbackOfOKMessageDialog(1);
+                }
+            }
+        </script>
+
+    </head>
         
-        //-->
-    </script>
-    <body>
-        <div style="text-align:left">
-            <table style="border-spacing:10px; border-collapse:collapse;" border="0">
+    <body onload="Document_OnLoad();">
+        <div style="text-align:left;">
+            <div id="PseudoDialogTitle"></div>
```

Comment:
@yusukemaegawa 

I edited this directly and added the following two commit.

- https://github.com/OpenTouryoProject/OpenTouryo/pull/237/commits/849fb09d715830f1aa049e2e9a6c3e6dd83cd3d2
- https://github.com/OpenTouryoProject/OpenTouryo/pull/237/commits/66ee77fe9646aebc7644846db6ddca75fc73e0c8